quirks/password-rules.json

```diff
@@ -198,6 +198,9 @@
     "spirit.com": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; required: [!@#$%^&*()];"
     },
+    "splunk.com": {
+        "password-rules": "minlength: 8; maxlength: 64; required: lower, upper, digit, [!@#$%&*_-+=<>];"
```

Comment:
This say, “requires a lower, or upper, or digit, or one of these from this character set.”

I’m guessing what you want is “requires a lower, and an upper, and a digit, and one of these from this character set”.

```
minlength: 8; maxlength: 64; required: lower; required: upper; required: digit; required: [!@#$%&*_-+=<>];
```

What do you think?

Reply:
perfect. yes. updated.